.gitignore

```diff
@@ -50,3 +50,22 @@ helm/**/*.tgz
 
 # Extracted strings for localization
 /libs/localization/messages.json
+
+# Ignore automatically openapi, generated schemas, types definitions, fragments types from codegen/autoSchemaFile/servers scripts
+/generated_files.tar.gz
+apps/api/src/api.graphql
+apps/web/graphql/fragmentTypes.json
+apps/web/graphql/schema.ts
+apps/gjafakort/api/src/types/schema.d.ts
+apps/gjafakort/web/graphql/schema.tsx
+apps/gjafakort/web/graphql/possibleTypes.json
+apps/air-discount-scheme/api.graphql
+apps/air-discount-scheme/web/graphql/schema.tsx
+apps/air-discount-scheme/web/graphql/possibleTypes.json
+apps/air-discount-scheme/backend/src/openapi.yaml
+apps/skilavottord/ws/src/app/api.graphql
+apps/judicial-system/api/src/openapi.yaml
+apps/application-system/api/src/openapi.yaml
+apps/reference-backend/src/openapi.yaml
+# libs/api/domains/cms/src/lib/generated/contentfulTypes.d.ts
+libs/api/schema/src/lib/schema.d.ts
```

Comment:
Can you change this to a wildcard.

Reply:
I added the changes for both `gitignore` and `clean-generated-files`. However, if someone decides to rename a generated file into `custom-name.d.ts` it will need to be added to the list but it shouldn't be the case